package.json

```diff
@@ -29,6 +29,7 @@
     "react-google-login": "^5.2.2",
     "react-i18next": "^11.8.13",
     "react-icons": "^4.2.0",
+    "react-loader-spinner": "^4.0.0",
     "react-redux": "^7.2.2",
```

Comment:
We have a loader in material ui, I suggest you use it instead of this one. 
Also uninstall redux